greenboot.spec

```diff
@@ -142,7 +145,7 @@ install -DpZm 0644 etc/greenboot/greenboot.conf %{buildroot}%{_sysconfdir}/%{nam
 %dir %{_prefix}/lib/%{name}/red.d
 %{_exec_prefix}/lib/motd.d/boot-status
 %{_tmpfilesdir}/greenboot-status-motd.conf
-%{_prefix}/etc/grub.d/*.cfg
+%{_sysconfdir}/grub.d/*.cfg
```

Comment:
This is an unrelated fix, it should be in a different commit/PR.

Reply:
I think you saw comparison between two commits, which made it seem unrelated.
Its actually changing the path from bootupd/static-config to grub.d